### PR TITLE
changed: drop unused OutputToAllSpeakers setting from database

### PR DESCRIPTION
**Pull request by notspiff**
and related settings classes.

unused according to comment.

### CODE REVIEW

**Comment by FernetMenta** [comment]
thanks!

File: xbmc/video/VideoDatabase.cpp [hunk 1]
```diff
@@ -5246,11 +5245,24 @@ void CVideoDatabase::UpdateTables(int iVersion)
...
+                "SubtitleDelay float, SubtitlesOn bool, Brightness float, Contrast float, Gamma float,"
+                "VolumeAmplification float, AudioDelay float, ResumeTime integer,"
+                "Sharpness float, NoiseReduction float, NonLinStretch bool, PostProcess bool,"
+                "ScalingMethod integer, DeinterlaceMode integer, StereoMode integer, StereoInvert bool, VideoStream integer)\n");
+    m_pDS->exec("INSERT INTO settingsnew (idFile, Deinterlace, ViewMode, ZoomAmount, PixelRatio, VerticalShift, AudioStream, SubtitleStream, SubtitleDelay, SubtitlesOn, Brightness, Contrast, Gamma, VolumeAmplification, AudioDelay, ResumeTime, Sharpness, NoiseReduction, NonLinStretch, PostProcess, ScalingMethod, DeinterlaceMode, StereoMode, StereoInvert, VideoStream) SELECT idFile, Deinterlace, ViewMode, ZoomAmount, PixelRatio, VerticalShift, AudioStream, SubtitleStream, SubtitleDelay, SubtitlesOn, Brightness, Contrast, Gamma, VolumeAmplification, AudioDelay, ResumeTime, Sharpness, NoiseReduction, NonLinStretch, PostProcess, ScalingMethod, DeinterlaceMode, StereoMode, StereoInvert, VideoStream FROM settings");
```
Comment (xhaggi) [inline xbmc/video/VideoDatabase.cpp]: if we only want to drop a column we could use `CREATE TABLE settingsnew AS SELECT <new_column_list> FROM`. This way we don't need to define the column type definition again.

File: xbmc/video/VideoDatabase.cpp [hunk 2]
```diff
@@ -5246,11 +5245,24 @@ void CVideoDatabase::UpdateTables(int iVersion)
...
+    m_pDS->exec("DROP TABLE settings");
+    m_pDS->exec("ALTER TABLE settingsnew RENAME TO settings");
```
Comment (xhaggi) [inline xbmc/video/VideoDatabase.cpp]: IIRC MySQL's syntax is without `TO` .. see https://dev.mysql.com/doc/refman/5.7/en/rename-table.html but SQLLite need the `TO`.

Comment (xhaggi) [reply xbmc/video/VideoDatabase.cpp]: Sorry for the noise the ALTER TABLE syntax supports both `RENAME [TO|AS] new_tbl_name`

File: xbmc/video/VideoDatabase.cpp (hunk 1 above)
Comment (notspiff) [reply xbmc/video/VideoDatabase.cpp]: learned something new.

**Review by xhaggi — APPROVED** [review]
Looks good.

**Pull request merged**